README.md

```diff
@@ -132,12 +116,9 @@ The easiest way to develop in docker while still using an IDE is to use VSCode a
     1. [VSCode](https://code.visualstudio.com/)
     1. [Docker](https://docs.docker.com/get-docker/)
 1. Open VSCode and install the following [plugins](https://code.visualstudio.com/docs/editor/extension-marketplace):
-    1. [VSCode Docker plugin](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker)
     1. [VSCode Remote Containers plugin](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
-1. Open this directory in a container by following [this guide](https://code.visualstudio.com/docs/remote/containers#_quick-start-open-an-existing-folder-in-a-container)
-    1. Due to a bug in the remote container plugin, you will need to refresh the window once it comes up. To do this, type `Ctrl+Shift+p` and type `Reload Window` and hit enter. Note that this will have to be repeated every time the container is rebuilt
-1. Once the folder is open in VSCode, you can build the project by running `Ctrl+Shift+B` to trigger a build, or `Ctrl+p` to open quick open, then type `task build` and hit enter
-1. You can run the project by following [this guide](https://code.visualstudio.com/docs/editor/debugging)
+1. Open this directory in a container by opening the `microstrain_inertial` directory in VSCode, clicking the green `><` icon in the bottom left corner of the window, and choosing `Reopen In Container`
```

Comment:
Sentence is a little confusing (could just be my inexperience with docker). Might be more clear to separate into sub-points:
1. Open this directory in a container:
    1. Open the `microstrain_inertial` directory in VSCode
    2. Click the green `><` icon in the bottom left corner of the window
    3. Choose `Reopen In Container`